README.md

```diff
@@ -559,8 +594,9 @@ while true; do
             case "${options[$selected]}" in
                 "Audio Server")
                     audio_server
-                    echo -e "You can login as CLI now.\nSelect 'Login CLI' to login!\n"
+                    echo -e "\nYou can login as CLI now.\nSelect 'Login CLI' to login!\n"
                     wait_for_key_press
+                    $HOME/chroot-xfce.sh
```

Comment:
Why are you calling the same script again? Also, if you want to call the same script again, please use `$0` as you can't know where the script resides, or even the name of the script.